src/test/java/com/unosquare/cvgenerator/util/MapperUtilTest.java

```diff
@@ -20,14 +21,15 @@
     private MapperUtil mapperUtil;
 
     @Test
-    public void map() {
+    public void map_singleObject() {
```

Comment:
this is interesting, @BrownMark curious to know what way you guys test mappings with model mapper

Reply:
Generally what i've done in the past is mock modelmapper and have specific tests for mappings. When it's convention based it might be worth skipping as your testing modelmapper itself, but then the problem is when a mapping moves from convention to specific the tests tend to be forgotten about.